### DISCUSSION TITLE
Technischer Walkthrough IdP und API in der Telematik RU (Referenzumgebung)

### DISCUSSION THREAD

**Discussion by ManuelB** [Digitaler-Impfnachweis/certification-apis#109] [Show and tell]
Ich hab mal unsere Lösung hier detailliert erklärt: https://youtu.be/oLrjF-OISO0 https://github.com/ere-health/dgc-ps-app

Das Video zeigt, wie man eine Checkout macht, die Anwendung startet, sich ein Impfzertifikat mit HTML Formular erstellt, sich einen Access Token holt, JSON validiert und einiges mehr.

Gruß
Manuel

**Comment by emelechow** [comment]
Hallo Manuel,

welche ID (VaccinationEntry:id) hast du bei den RU Tests verwendet?


Schöne Grüße
Eugen

**Reply by ManuelB** [reply]
@emelechow typischweise eine Betriebsstättennummer eines Arztes: https://de.wikipedia.org/wiki/Betriebsst%C3%A4ttennummer

**Reply by emelechow** [reply]
Verwenden wir auch, bekommen aber auf RU "not whitelisted" Fehler

**Reply by ManuelB** [reply]
Wir haben das vor circa 4 Wochen gemacht, möglicherweise wurde das mittlerweile etwas geändert.

**Reply by emelechow** [reply]
Supportanfrage habe ich gestellt - leider noch ohne Antwort. Habt ihr produktive BSNR verwendet?

**Reply by ManuelB** [reply]
Nicht auf der RU. Wir haben mittlerweile aber viele Praxen, die unsere Lösung im produktiven Betrieb nutzen.

**Reply by emelechow** [reply]
Dann versuchen wir das in unserer Praxis! Danke für schnelles Feedback!
